### ISSUE TITLE
Ensure list items are found in a list container

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/onrr.gov-site#2702]
OCIO Violation: Ensure list items are found in a list container

Severity: 3

Occurrence: 92%

Priority: 5.8

Note: This A does not have a ul element (without an ARIA-assigned role); ol element (without an ARIA-assigned role); an element with a role set to the value: list as a parent; or a ul element (without an ARIA-assigned role), ol element (without an ARIA-assigned role) or element with a role set to the value 'list' with an aria-owns attribute set to the ID of the element in the same DOM

Description: Each of these indicates the slug where it happens in the location of the violation.

See attached file for list of occurances. [https://app.zenhub.com/files/210892300/f95fa0a4-c8b8-48c4-bdb6-99f3399d097b/download](https://app.zenhub.com/files/210892300/f95fa0a4-c8b8-48c4-bdb6-99f3399d097b/download)

**Comment by eelzi-ONRR** [comment]
2024 OCIO report:
Severity: 3

Occurrence: 94%

Priority: 6.6

**Comment by greensomes** [comment]
I've reviewed the attached document and the instances have been resolved.
